### PR TITLE
W-10647418

### PR DESCRIPTION
**Pull request by PabloCamino**
Clarification on local behavior of RTF restore after working on referenced investigation
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000lk5msYAA/view

### CODE REVIEW

File: modules/ROOT/pages/troubleshoot-guide.adoc [hunk 1]
```diff
@@ -334,3 +335,18 @@ To renew the certificate, use the `rtfctl appliance renew-expired-client-cert` c
...
+
+== Troubleshoot Inconsistencies After Backing Up and Restoring Rutime Fabric
+
+After a restoring a cluster from a backup, you may see some inconsistencies in Runtime Manager, for example: 
+
+* An application you deleted shows as running
+* A recently deployed application is listed with 0 replicas
+* A deployment's replica count does not match the number of replicas you specified in Runtime Manager
+
+In each of these circumstances, verify that the deployments appear in *Applying* status and that a corresponding API call reveals that the `status` and `desiredState` fields do not match. The root cause is explained in xref:manage-backup-restore.adoc#back-up-and-restore-process-does-not-affect-the-control-plane[The Back Up and Restore Process Does Not Affect the Control Plane].
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/troubleshoot-guide.adoc]:
```suggestion
In each of these circumstances, verify that the deployments appear in *Applying* status and that a corresponding API call reveals that the `status` and `desiredState` fields do not match. See xref:manage-backup-restore.adoc#back-up-and-restore-process-does-not-affect-the-control-plane[The Back Up and Restore Process Does Not Affect the Control Plane] for an explanation of the root cause.
```

**Comment by IsaacEldridge** [comment]
@PabloCamino Thanks for the contribution--I made some edits, can you please review for accuracy?

**Review by PabloCamino — COMMENTED** [review]
Changes look goos, I just cleared a typo

File: modules/ROOT/pages/troubleshoot-guide.adoc [hunk 2]
```diff
@@ -334,3 +335,17 @@ To renew the certificate, use the `rtfctl appliance renew-expired-client-cert` c
...
+
+== Troubleshoot Inconsistencies After Backing Up and Restoring Rutime Fabric
+
+After restoring a cluster from a backup, you may see some inconsistencies in Runtime Manager, for example: 
+
+* An application you deleted shows as running
+* A recently deployed application is listed with 0 replicas
+* A deployment's replica count does not match the number of replicas you specified in Runtime Manager
+
+In each of these circumstances, verify that the deployments appear in *Applying* status and that a corresponding API call reveals that the `status` and `desiredState` fields do not match. See xref:manage-backup-restore.adoc#back-up-and-restore-process-does-not-affect-the-control-plane[The Back Up and Restore Process Does Not Affect the Control Plane] for an explanation of the root cause.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/troubleshoot-guide.adoc]:
```suggestion
In each of these circumstances, verify that the deployments status appears as *applying* and that a corresponding API call reveals that the `status` and `desiredState` fields do not match. See xref:manage-backup-restore.adoc#back-up-and-restore-process-does-not-affect-the-control-plane[The Back Up and Restore Process Does Not Affect the Control Plane] for an explanation of the root cause.
```

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**